cmd/po-docgen/operator.go

```diff
@@ -0,0 +1,226 @@
+	if ok {
+		retValue := ""
+		if exprIdent.Obj.Kind == ast.Con {
+			valSpec := exprIdent.Obj.Decl.(*ast.ValueSpec)
+			for _, value := range valSpec.Values {
+				retValue += unquoteLiteral(value.(*ast.BasicLit).Value)
+			}
+		}
+		return retValue
+	}
+
+	exprBinaryOp, ok := expr.(*ast.BinaryExpr)
+	if ok {
+		operandX := resolveConstStringExpr(exprBinaryOp.X)
+		operandY := resolveConstStringExpr(exprBinaryOp.Y)
+		if exprBinaryOp.Op == token.ADD {
+			return operandX + operandY
+		}
+		return fmt.Sprint("unhandled OP", exprBinaryOp.Op)
+	}
+
+	return ""
+}
+
+func operatorCodeToDoc(paths []string) []FlagDoc {
+	astPkg := fileToASTPackage(paths)
+	flagDocs := make([]FlagDoc, 0)
+
+	for _, file := range astPkg.Files {
+		fmt.Println(file.Name.Name)
+		for _, decl := range file.Decls {
+			funcdecl, ok := decl.(*ast.FuncDecl)
+			if ok {
+				if funcdecl.Name.Name == "init" {
+					for _, statement := range funcdecl.Body.List {
+						exprStatement, ok := statement.(*ast.ExprStmt)
+						if !ok {
+							continue
+						}
+						exprCall, ok := exprStatement.X.(*ast.CallExpr)
+						if !ok {
+							continue
+						}
+
+						if exprCall.Fun.(*ast.SelectorExpr).X.(*ast.Ident).Name == "flagset" {
+							var argName, argDefaultValue, argDescription string
+							if exprCall.Fun.(*ast.SelectorExpr).Sel.Name == "StringVar" {
+								argName = unquoteLiteral(exprCall.Args[1].(*ast.BasicLit).Value)
+								argDefaultValue = resolveConstStringExpr(exprCall.Args[2])
+								if len(argDefaultValue) == 0 {
+									argDefaultValue = "\"\""
+								}
+								argDescription = resolveConstStringExpr(exprCall.Args[3])
+
+							} else if exprCall.Fun.(*ast.SelectorExpr).Sel.Name == "Var" {
+								argName = unquoteLiteral(exprCall.Args[1].(*ast.BasicLit).Value)
+								argDefaultValue = "N/A"
+								argDescription = resolveConstStringExpr(exprCall.Args[2])
+
+							} else if exprCall.Fun.(*ast.SelectorExpr).Sel.Name == "DurationVar" {
+								argName = unquoteLiteral(exprCall.Args[1].(*ast.BasicLit).Value)
+								argDefaultValue = resolveDurationExpr(exprCall.Args[2])
+								argDescription = resolveConstStringExpr(exprCall.Args[3])
+
+							} else if exprCall.Fun.(*ast.SelectorExpr).Sel.Name == "BoolVar" {
+								argName = unquoteLiteral(exprCall.Args[1].(*ast.BasicLit).Value)
+								argDefaultValue = resolveBoolExpr(exprCall.Args[2])
+								argDescription = resolveConstStringExpr(exprCall.Args[3])
+
+							}
+							flagdoc := FlagDoc{
+								name:         argName,
+								defaultValue: argDefaultValue,
+								description:  argDescription,
+							}
+
+							flagDocs = append(flagDocs, flagdoc)
+
+						}
+
+					}
+				}
+
+			}
+
+		}
+	}
+	return flagDocs
+
+}
+
+func printOperatorDocs(paths []string) {
+	fmt.Print(`---
+title: "Operator CLI Flags"
+description: "Lists of possible arguments passed to operator executable."
+date: 2021-06-18T14:12:33-00:00
+draft: false
+images: []
+menu:
+	docs:
+	parent: "operator"
```

Comment:
I think that the indentation is wrong.

```suggestion
  docs:
    parent: "operator"
```